docs/api-and-integration/tentacle.exe-command-line/update-trust.md

```diff
@@ -0,0 +1,24 @@
+---
+title: update-trust
+description: Replaces the trusted Octopus Server thumbprint of any matching polling or listening registrations with a new thumbprint to trust
+---
+
+Replaces the trusted Octopus Server thumbprint of any matching polling or listening registrations with a new thumbprint to trust
```

Comment:
Minor nitpick for next time the tool is being updated. It'd be good to end with a colon. ...a new thumbprint to trust: